test/test_images/probe_helper/probe_helper.go

```diff
@@ -243,6 +246,13 @@ func (ph *ProbeHelper) forwardFromProbe(ctx context.Context, brokerClient cloude
 			default:
 				return logNACK(ctx, "Probe forwarding failed, unrecognized cloud storage probe subject: %v", event.Subject())
 			}
+		case CloudSchedulerSourceProbeEventType:
+			channelID = cloudSchedulerSourceProbeChannelID
```

Comment:
If we receive a probe requests while there is another being processed, the new request will fail due to the duplicated channel ID.

I don't think we should rely on the channel approach for the scheduler any more. How about this - assuming a scheduler source running with a tick period of say, 1min. The receiver updates the lastUpdatedTimestamp. When we receive a probe, the elapsed time since lastUpdatedTimestamp should be within a minute (like the liveness probe).

Reply:
Good idea. Done.